CONTRIBUTING.md

```diff
@@ -19,15 +19,13 @@ Shared tsconfig for use across all projects.
 First, install all the dependencies (using [Volta](https://volta.sh/) is highly recommended):
 
 ```shell
-yarn install
+yarn install --frozen-lockfile
```

Comment:
Haha I was wondering about the difference between `yarn install` and `yarn install --frozen-lockfile`, and found a github thread you commented on from 2017
https://github.com/yarnpkg/yarn/issues/4147#issuecomment-321774630